SumoLogic.Logging.Nuget.bat

```diff
@@ -1,6 +1,4 @@
 @echo off
-echo "Building Libraries"
-msbuild /p:Configuration=Release SumoLogic.Logging.CI.csproj 
-echo CREATING NUGET PACKAGES ...
-echo ======================================
-msbuild SumoLogic.Logging.CI.csproj /t:MakeNugetPackages /v:m /p:Configuration=Release
+dotnet pack --output %~dp0SumoLogic.Logging.Nuget --configuration Release SumoLogic.Logging.Common/SumoLogic.Logging.Common.csproj

```

Comment:
I assume these should each output to a different nuget package file, not all to `SumoLogic.Logging.Nuget`?